src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,94 @@
+package lotto.domain;
+
+public class LottoResult {
+
+	private static final int FIRST_RANK_EARNING = 2000000000;
```

Comment:
Java 7부터는 `100_000_000`처럼 숫자에 밑줄을 사용하여 가독성을 높일 수 있어요.

```suggestion
	private static final int FIRST_RANK_EARNING = 2_000_000_000;
```